src/utils/jsHelper/helper.tsx

```
@@ -362,97 +380,97 @@ export function getSubmmitedDoc(userData, document) {
+		// Check if value is not in the condition values
+		return !conditionVals.includes(val);
+	default:
+		// Return false for unrecognized conditions
+		return false;
+	}
 }
 export function getIncomeRangeValue(annualIncome: string): string | undefined {
-  const income = parseFloat(annualIncome);
-  if (isNaN(income)) return "";
+	const income = parseFloat(annualIncome);
+	if (isNaN(income)) return '';
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)